.github/workflows/stale.yml

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Close stale issues and PRs'
+on:
+  #schedule:
+  #  - cron: '30 3 * * *'
+  workflow_dispatch:
+    inputs:
+      dryRun:
+        description: 'Dry Run'
+        required: true
+        default: true
+        type: boolean
+      operationsPerRun:
+        description: 'Max GitHub API operations'
+        required: true
+        default: 30
+        type: number
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          debug-only: ${{ inputs.dryRun }}
+          operations-per-run: ${{ inputs.operationsPerRun }}
+          days-before-stale: 90
+          days-before-close: -1
```

Comment:
Why don’t we close them as well after another 30 days? It won’t be close of a reviewer or the contributor reacts. I am saying this because without closing them, it does not really with reducing the backlog.

Reply:
I'd like to in a future PR. Since we can't test the action until it's committed, I wanted to hold off on doing anything "destructive" like closing a bunch of PRs :) 

After testing this manually with the dry-run option, I'll raise a PR to add the schedule and close PRs

Reply:
I also think that having a dry run first and see that the right PRs would be flagged is a good approach